modules/installation-osp-setting-worker-affinity.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
+
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+// TODO: Verify supportability of manifest modification.
+
+[id="installation-osp-setting-worker-affinity_{context}"]
+= Setting compute machine affinity
+
+Optionally, you can set the affinity policy for compute machines during installation.
+
+[TIP]
+====
+The installer does not select an affinity policy for compute machines by default. 
+
+Control plane machines are created with a soft anti-affinity policy. 
```

Comment:
Openstack calls it "soft-anti-affinity" with two dashes

Reply:
Will harmonize with OSP, in that case.

Reply:
```suggestion
Control plane machines are created with a soft-anti-affinity policy. 
```

Reply:
```suggestion
Control plane machines are created with a `soft-anti-affinity` policy. 
```